lib/rust/metamodel/src/java/mod.rs

```diff
@@ -0,0 +1,233 @@
+//! Representation of datatype definitions in the Java typesystem.
+
+pub mod bincode;
+mod from_meta;
+#[cfg(feature = "graphviz")]
+pub mod graphviz;
```

Comment:
is our file formatter supporting pragmas on imports? Was this file formatted with it?

Reply:
This is configured by the [inline_attribute_width](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#inline_attribute_width) parameter, which we currently have disabled (set to `0`). Should I change it?

Reply:
Oh--yes, it handles it correctly.